code/game/objects/items/weapons/grenades/atmosgrenade.dm

```diff
@@ -47,7 +47,7 @@
 	for(var/turf/T in view(freeze_range, loc))
 		if(isfloorturf(T))
 			var/turf/simulated/F = T
-			F.MakeSlippery(TURF_WET_PERMAFROST)
+			F.MakeSlippery(TURF_WET_ICE)
```

Comment:
This code isn't great.

It loops through all turfs in range, but then does an `isfloorturf` check on them.

This should definitely just loop through floor turfs, only, to begin with.

Reply:
It now loops through only floor turfs